modules/programs/waybar.nix

```diff
@@ -199,7 +199,7 @@ in {
 
     systemd.target = mkOption {
       type = str;
-      default = "graphical-session.target";
+      default = "tray.target";
```

Comment:
Isn't this racy?
Other services could also start as they might be wanted by `tray.target`, before waybar has even initialized.

Reply:
I guess short of having those services instead depend on `waybar.service`, you'd have to create a `tray-pre.target`, which would make sense to me if people are pinning tasks that go into a tray to `tray.target`, and they as such need a tray service to pre-exist to put it in.